packages/core/package.json

```diff
@@ -261,19 +261,19 @@
                     },
                     "additionalProperties": false
                 },
-                "aws.amazonQ.includeSuggestionsWithCodeReferences": {
+                "aws.amazonQ.showInlineCodeSuggestionsWithCodeReferences": {
```

Comment:
Out of curiosity why are we still declaring these in core when they are only relevant to q now? Is it for tests?

Reply:
[https://github.com/aws/aws-toolkit-vscode/blob/master/docs/arch_develop.md?plain=1#L60
and
](https://github.com/aws/aws-toolkit-vscode/blob/feature/standalone/docs/arch_develop.md#contributes-and-settings)